### DISCUSSION TITLE
Pouvoir hide les barres supérieures

### DISCUSSION THREAD

**Discussion by mirobane** [skarab42/marv#21] [Ideas]
coucou !

l'idée est de pouvoir avoir dans l'interface sur mobile la possibilité de cacher l'une ou l'autre voir les deux barres supérieures de l'interface de marv' pour optimiser la place d'affichage sur les périphériques a petit écran.

**Comment by skarab42** [comment]
Je ferais un travail sur les interface tactile en v1.x avec possibilité de switch de panneau avec un slide ;)
Voir aussi #30 #31
